ynput/ayon-core
# Maya: Implement backward compatibility for version up workfile

## Changelog Description
Implement backward compatibility for version up workfile in maya menu for just in case the related setting being turned off or the user has been in the lower addon version.

## Additional info
Paragraphs of text giving context of additional technical information or code examples.

## Testing notes:
1. Build the maya addon with this branch
2. Install addon
3. Launch Maya
4. AYON menu should show all the plugins as expected.
